services/version.js

```diff
@@ -237,6 +237,7 @@ function rangeStart(v) {
  * @param {string} [options.prefix]  - The prefix to display on the message, such as ">=", "v", overrides the default behavior of using addv
  * @param {string} [options.postfix] - The postfix to display on the message, such as "tested"
  * @param {Function} [options.versionFormatter=versionColor] - The function to use to format the color of the badge based on the version number
+ * @param {boolean} [options.forcePrerelease] - Whether to force the badge to be considered a prerelease for cases where upstream API indicates a prerelease version but it is not semantically a prerelease
```

Comment:
I'd probably just call this `isPrerelease`

The way I'd view this is not so much that we're allowing platforms to indicate that a semantically stable release is actually a pre-release. More just that it allows a platform to explicitly indicate a release is a pre-release rather than us having to try and infer it.

Reply:
Its probably for the better, our goal is to serve from upstream rather then infer our own conclusion.
Changed with [c6714e3](https://github.com/badges/shields/pull/10656/commits/c6714e350119592275e86fd61059b8c179dcc029)